app/models/transformation_mapping.rb

```diff
@@ -3,7 +3,13 @@ class TransformationMapping < ApplicationRecord
 
   validates :name, :presence => true, :uniqueness => true
 
+  virtual_has_many :service_template_transformation_plans
+
   def destination(source)
     transformation_mapping_items.find_by(:source => source).try(:destination)
   end
+
+  def service_template_transformation_plans
```

Comment:
The actual logic does not filter for `ServiceTemplateTransformationPlan` although in reality they are likely always transformation plan. Should we relax the method name just `service_templates`?